uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/my-leave/components/my-leave.js

```diff
@@ -10,8 +10,7 @@ define([
     }],
     controllerAs: 'myleave',
     controller: ['$log', '$scope', '$uibModal', 'settings', function ($log, $scope, $modal, settings) {
-      console.log('CRM id of the currently logged in user: ', this.contactId);
-      $log.debug('Component: my-leave');
+      $log.debug('Component: my-leave', this.contactId);
```

Comment:
I think `this.contactId` can be removed, there is no point at this stage to keep logging it